src/HomePage/_home.scss

```diff
@@ -1,26 +1,95 @@
 .main {
   width: 100vw;
   overflow-x: hidden;
+
+  .promo-banner {
```

Comment:
Changes ini bsa dibawa ke styles.scss aja ga? Abisnya kan promo-banner dipakai di multiple pages, kalau bisa settingannya di styles.scss aja

Reply:
Ohh gitu, bisa jg sih.
Sbenarnya kalau thats the case, kalau gtu bkin promo-banner jadi absolute khusus buat home jg gpp